CHANGELOG.asciidoc

```diff
@@ -215,6 +215,7 @@ https://github.com/elastic/beats/compare/v5.3.2...v5.4.0[View commits]
 - Fix potential Elasticsearch output URL parsing error if protocol scheme is missing. {pull}3671[3671]
 - Downgrade Elasticsearch per batch item failure log to debug level. {issue}3953[3953]
 - Make `@timestamp` accessible from format strings. {pull}3721[3721]
+- Fixing configuration documentation for kubernetes processor {pull}4313[4313]
```

Comment:
Sorry, I just noticed this line went to an already released version, could you move it to the HEAD? also it would be great if you rename `Fixing` -> `Fix`